src/libraries/System.IO.FileSystem/tests/FileStream/LockUnlock.cs

```diff
@@ -72,6 +71,22 @@ public void Lock_Unlock_Successful(long fileLength, long position, long length)
             }
         }
 
+        [Theory]
+        [InlineData(FileAccess.Read)]
+        [InlineData(FileAccess.Write)]
+        [InlineData(FileAccess.ReadWrite)]
+        [PlatformSpecific(~TestPlatforms.OSX)]
+        public void Lock_Unlock_Successful_AlternateFileAccess(FileAccess fileAccess)
+        {
+            string path = GetTestFilePath();
+            File.WriteAllBytes(path, new byte[100]);
+
+            using FileStream fs = File.Open(path, FileMode.Open, fileAccess);
```

Comment:
Should we perform an additional action with the file to make sure the `fileAccess` worked?

Reply:
I don't think so given that this file doesn't contain tests for that and judging by the filename is not meant to do so. I would expect that there exists dedicated tests that verify FileAccess.

Reply:
As per offline discussion, this should be resolved by the tests added in https://github.com/dotnet/runtime/pull/44185/commits/0d2f64ac6a9532eead90626b0467a842f7c05520, the "additional action" is to try to Lock/Unlock the file with RemoteExecutor.